From a271effaaf8bcaf86f8175e5ad96969757d4c4fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Wed, 12 Aug 2020 16:35:07 +0200
Subject: [PATCH] syncthing: move home directory to /srv

Turris uses /srv and has support for easy mounting of drives to that
location.
---
 utils/syncthing/files/etc/config/syncthing | 2 +-
 utils/syncthing/files/etc/init.d/syncthing | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/syncthing/files/etc/config/syncthing b/utils/syncthing/files/etc/config/syncthing
index 291675d40..75e04caa5 100644
--- a/utils/syncthing/files/etc/config/syncthing
+++ b/utils/syncthing/files/etc/config/syncthing
@@ -10,7 +10,7 @@ config syncthing 'syncthing'
 	#   be readable/writable by the user specified in
 	#   'option user'.
 	# Consult syslog if things go wrong.
-	option home '/etc/syncthing'
+	option home '/srv/syncthing'
 
 	# Changes to "niceness"/macprocs are not picked up by "reload_config"
 	#   nor by "restart": the service has to be stopped/started
diff --git a/utils/syncthing/files/etc/init.d/syncthing b/utils/syncthing/files/etc/init.d/syncthing
index f77aad8ae..9fafd6913 100755
--- a/utils/syncthing/files/etc/init.d/syncthing
+++ b/utils/syncthing/files/etc/init.d/syncthing
@@ -26,7 +26,7 @@ start_service() {
 
   config_get user syncthing user 'root'
   config_get gui_address syncthing gui_address "http://127.0.0.1:8384"
-  config_get home syncthing home "/etc/syncthing"
+  config_get home syncthing home "/srv/syncthing"
 
   # For backwards compatibility
   IDX_DB=$(readlink -n "$home"/index-v0.14.0.db)
-- 
2.28.0

